### DISCUSSION TITLE
Want to zoom in when chart load in react js functional component

### DISCUSSION THREAD

**Discussion by zeeshan-shabbir** [tradingview/lightweight-charts#1017] [Q&A]
![Screenshot_2022-02-15-21-22-53-554_com whatsapp](https://user-images.githubusercontent.com/80969033/154104185-e6ca90b2-5161-4149-afcc-8f3e6ee69a52.jpg)

**Comment by zeeshan-shabbir** [comment]
I wnat it to zoom in i dont know how to do this timescales in reacf function component

**Comment by edew** [comment]
> I wnat it to zoom in

What do you mean by "zoom in"?

The `setVisibleRange` API can be used to change the time range that is shown: https://tradingview.github.io/lightweight-charts/docs/api/interfaces/ITimeScaleApi#setvisiblerange.

**Comment by zeeshan-shabbir** [comment]
> > I wnat it to zoom in
> 
> What do you mean by "zoom in"?
> 
when my graph render it show timescale in year i want it to show as per day so the candle is more visible i hope you will understand and help me.

**Comment by zeeshan-shabbir** [comment]
> 
> The `setVisibleRange` API can be used to change the time range that is shown: https://tradingview.github.io/lightweight-charts/docs/api/interfaces/ITimeScaleApi#setvisiblerange.

i dont known from and to values as data is come from api

**Comment by timocov** [comment]
> i dont known from and to values as data is come from api

The library neither since it comes _from you_.
